lvsfunc/util.py

```diff
@@ -102,6 +103,45 @@ def get_prop(frame: vs.VideoFrame, key: str, t: Type[T]) -> T:
     return prop
 
 
+def select_frames(clips: Union[vs.VideoNode, List[vs.VideoNode]],
+                  indicies: Union[List[int], List[Tuple[int, int]]]) -> vs.VideoNode:
+    """
+    Select frames from one or more clips at specified indices.
+
+    :param clips:   A clip or a list of clips to select the frames from
+    :param indexes: Indices of frames to select.
```

Comment:
you want `indices`, not `indicies` or `indexes` (misspelled throughout the function)

Reply:
Writing code while drunk is a bad idea